test/evp_kdf_test.c

```diff
@@ -1629,6 +1629,148 @@ static int test_kdf_ss_kmac(void)
     return ret;
 }
 
+static 
+OSSL_PARAM *construct_snmpkdf_params(char *digest, unsigned char *eid, size_t eid_len,
+                                      unsigned char *password, size_t password_len)
+
+{
+    OSSL_PARAM *p = OPENSSL_malloc(sizeof(OSSL_PARAM) * 4);
```

Comment:
unless it is necessary I would have done the failure tests as evpkdf_snmp.txt entries. You can use the Result tag e..g.
````
Result = KDF_DERIVE_ERROR
````

Reply:
moved tests that could be done in evpkdf_snmp.txt